scripts/bootstrap.py

```diff
@@ -0,0 +1,271 @@
+        color_blue = Fore.BLUE
+    except ImportError:
+        pass
+
+
+def make_key(key_type: str) -> Tuple[str, str]:
+    secret = subprocess.run([jcli, 'key', 'generate', '--type={}'.format(
+        key_type)], capture_output=True, check=True).stdout
+    public = subprocess.run([jcli, 'key', 'to-public'],
+                            input=secret, capture_output=True, check=True).stdout
+    return secret.decode('utf-8').strip(), public.decode('utf-8').strip()
+
+
+def make_key_and_address(key_type: str) -> Tuple[str, str, str]:
+    secret, public = make_key(key_type)
+    address = subprocess.run([jcli, 'address', 'account', ADDRESS_TYPE, public],
+                             capture_output=True, check=True).stdout.decode('utf-8').strip()
+    return secret, public, address
+
+
+def sign_certificate(certificate: bytes, secret_key: str) -> bytes:
+    key_file = tempfile.NamedTemporaryFile(mode='w')
+    key_file.write(secret_key)
+    key_file.flush()
+    certificate_signed = subprocess.run([jcli, 'certificate', 'sign', '--key', key_file.name],
+                                        input=certificate, capture_output=True, check=True).stdout
+    return certificate_signed
+
+
+faucet_secret_key, faucet_public_key, faucet_address = make_key_and_address(
+    'Ed25519Extended')
+fixed_secret_key, fixed_public_key, fixed_address = make_key_and_address(
+    'Ed25519Extended')
+leader_secret_key, leader_public_key = make_key('Ed25519')
+vrf_secret_key, vrf_public_key = make_key('Curve25519_2HashDH')
+kes_secret_key, kes_public_key = make_key('SumEd25519_12')
+
+stake_key_secret = faucet_secret_key
+stake_key_public = faucet_public_key
+
+stake_pool_certificate_unsigned = subprocess.run([jcli, 'certificate', 'new', 'stake-pool-registration', '--management-threshold', '1', '--start-validity', '0', '--owner', leader_public_key, '--kes-key',
+                                                  kes_public_key, '--tax-fixed', '10', '--tax-limit', '1000000000', '--tax-ratio', '1/10', '--reward-account', faucet_address, '--vrf-key', vrf_public_key], capture_output=True, check=True).stdout
+stake_pool_certificate = sign_certificate(
+    stake_pool_certificate_unsigned, leader_secret_key)
+
+stake_pool_id = subprocess.run([jcli, 'certificate', 'show', 'stake-pool-id'],
+                               input=stake_pool_certificate, capture_output=True, check=True).stdout.decode('utf-8').strip()
+
+stake_delegation1_unsigned = subprocess.run(
+    [jcli, 'certificate', 'new', 'stake-delegation', faucet_public_key, stake_pool_id], capture_output=True, check=True).stdout
+stake_delegation1 = sign_certificate(
+    stake_delegation1_unsigned, faucet_secret_key)
+
+stake_delegation2_unsigned = subprocess.run(
+    [jcli, 'certificate', 'new', 'stake-delegation', fixed_public_key, stake_pool_id], capture_output=True, check=True).stdout
+stake_delegation2 = sign_certificate(
+    stake_delegation2_unsigned, fixed_secret_key)
+
+genesis_yaml = '''\
```

Comment:
We could use [pydantic](https://pydantic-docs.helpmanual.io/) for this

Reply:
We can achieve pretty much the same thing by just passing a `kwargs` dictionary. Or use [template strings](https://docs.python.org/3.4/library/string.html#template-strings).